homeassistant/components/torque/config_flow.py

```diff
@@ -0,0 +1,86 @@
+"""Config flow for Torque integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_EMAIL, CONF_NAME
+from homeassistant.core import DOMAIN as HOMEASSISTANT_DOMAIN
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.issue_registry import IssueSeverity, async_create_issue
+
+from .const import DEFAULT_NAME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+ISSUE_PLACEHOLDER = {"url": "/config/integrations/dashboard/add?domain=torque"}
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_EMAIL): cv.string,
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
```

Comment:
Setting names are only allowed for helpers

Reply:
I'm not entirely sure what you mean by this, I was trying to replicate [the usage here](https://www.home-assistant.io/integrations/torque#name)? Should I remove this line completely? 🤔

Reply:
Yep! There are still some other config flows that have settable name, but we're slowly getting rid of them :)